lib/rubocop/cop/rspec/let_before_examples.rb

```diff
@@ -44,9 +44,7 @@ class LetBeforeExamples < Cop
         def on_block(node)
           return unless example_group_with_body?(node)
 
-          _describe, _args, body = *node
-
-          check_let_declarations(body)
+          check_let_declarations(node.body) if node.body.begin_type?
```

Comment:
I’m not sure what this (`node.body.begin_type?`) means.

Reply:
Single-line block looks like this:
```
(block
  (send nil :include_examples
    (str "special user"))
  (args)
  (block
    (send nil :let
      (sym :foo))
    (args)
    (send nil :bar)))
```

while multi-line have a (begin)
```
(block
  (send nil :include_examples
    (str "special user"))
  (args)
  (begin
    (block
      (send nil :let
        (sym :foo))
      (args)
      (send nil :bar))
    (block
      (send nil :let
        (sym :bar))
      (args)
      (send nil :baz))))
```

So I use this fact to determine if the body of the block have more than one statement. I can extract it in appropriately named method for better readability